ui/raidboss/data/05-shb/raid/e7n.js

```diff
@@ -102,7 +102,7 @@
       condition: Conditions.targetIsYou(),
       suppressSeconds: 3,
       infoText: function(data) {
-        data.colorCount = data.colorCount + 1 || 1;
+        data.colorCount = data.colorCount + 1 || 0;
```

Comment:
Belatedly, I don't think I realized the NaN shenanigans (sheNaNigans??) going on here before to make this statement work out properly when data.colorCount is undefined.

Reply:
Oof. I suppose I should have separated the counter from the definition, you're right. I'll fix that if/when there's other stuff I have to touch in here.